lib/tasks/publishing_api.rake

```diff
@@ -24,4 +24,12 @@ namespace :publishing_api do
       end
     end
   end
+
+  desc "Unpublish special routes"
+  task :unpublish_special_routes do
```

Comment:
Could we rename this to `unpublish_business_readiness_special_routes` since it only does those?

Reply:
👍 see what you think of the name in 7885a65. The routes aren't both related to business readiness so I've gone for a slightly more verbose and descriptive name without using the entire route names because that would become rather cumbersome.